google/cloud/pubsub/internal/tracing_message_callback.cc

```diff
@@ -50,13 +53,17 @@ class TracingMessageCallback : public MessageCallback {
     auto span =
         internal::MakeSpan(subscription_id_ + " process",
                            {{sc::kMessagingSystem, "gcp_pubsub"}}, options);
-
+    if (enable_otel_) {
+      m.ack_handler = MakeTracingExactlyOnceAckHandler(std::move(m.ack_handler),
+                                                       m.subscribe_span);
+    }
     child_->user_callback(std::move(m));
     span->End();
   }
 
   std::shared_ptr<MessageCallback> child_;
   std::string subscription_id_;
+  bool enable_otel_;
```

Comment:
We already checked this. It's value can only be `true`: https://github.com/googleapis/google-cloud-cpp/blob/da557d7e77b0160dcf0cb03f843afd89e97ba3cf/google/cloud/pubsub/internal/subscription_session.cc#L46-L48

Reply:
You're right. Removed.